Project-Hotpot/Project-Hotpot/ViewControllers/HomeViewController.swift

```diff
@@ -34,6 +34,15 @@ class HomeViewController: UIViewController {
         }
     }
     
+    func presentAlertController(title: String, message: String, buttonTitle: String) {
```

Comment:
Think about function and class names a bit more what they convey. Function names are always verbs describing an action like 'showAlert()'. Class names are always noun describing what entity this class represents. Here presentAlertController does not seem to provide good readability. Consider 'showAlert' or 'presentAlert'.
The example is right in the impl of this method. Look at the UIAlertController ( a class) and its methods 'addAction'